plane/src/typed_socket/client.rs

```diff
@@ -88,15 +88,14 @@ impl<T: ChannelMessage> TypedSocketConnector<T> {
 
 /// Turns a URL (which may have a token) into a request. If the URL contains a token, it is
```

Comment:
this comment is no longer accurate right? since the token is removed at the AuthorizedUrl::From stage?

Reply:
good catch, thanks!